lib/travis/config/defaults.rb

```diff
@@ -57,18 +57,5 @@ def initialize(*)
       super
       load_urls
     end
-
-    # Wild monkeypatch to backport travis-config v1.0.x resource url loading.
-    # Needed for enterprise.
-    class Docker
-      def load
-        config = compact(
-          database: Database.new.config,
-          logs_database: Database.new(prefix: :logs).config,
-          amqp: Url.parse(ENV['TRAVIS_RABBITMQ_URL'] || ENV['RABBITMQ_URL']).to_h.compact,
-          redis: { url: ENV['TRAVIS_REDIS_URL'] || ENV['REDIS_URL'] }.compact
-        )
-      end
-    end
```

Comment:
👍 Super cool. Since this should be in the updated travis-config we should be good. If it doesn't work we'll catch it when doing HA testing.